src/Sylius/Bundle/CoreBundle/Command/InstallSampleDataCommand.php

```diff
@@ -33,6 +34,7 @@ protected function configure(): void
 The <info>%command.name%</info> command loads the sample data for Sylius.
 EOT
             )
+            ->addOption('fixture-suite', 's', InputOption::VALUE_OPTIONAL, 'Load specified fixture suite during install', null)
```

Comment:
Won't it be better to have `default` as a default value instead of null? We will reduce one if statement `$suite ?? 'default'`, while `null !== $suite` could be changed to `'default' !== $suite`. 

This will mean, that we cannot load not interactively default fixtures. However, I'm not convinced about this change anyway. The feature is useful, but consequences may be huge, so I would not load it.